docs/HowToFAQ/Errors-Rewriter.md

```diff
@@ -0,0 +1,343 @@
+
+<!-- %check-resolve %default %useHeadings -->
+
+<!-- ./DafnyCore/Rewriters/TimeLimitRewriter.cs -->
+
+## **Warning: timeLimitMultiplier annotation overrides " + name + " annotation**
+
+```ddafny
+method {:timeLimitMultiplier 10} {:timeLimit 5} m() {}
+```
+
+The {:timeLimitMultiplier n} attribute tells the prover to use a time limit that is the given number
+times the time limit otherwise specified by the options. The {:timeLimit n} attribute simply sets a new
+value for the time limit. It does not make sense to have both attributes on the same declaration. 
+One or the other should be removed.
```

Comment:
If both attributes are present, what is the actual behavior? The warning text suggests to me that `timeLImitMultiplier` takes precedence, but the explanation doesn't say one way or the other. Since this is marked as a warning and not an error, I think we should be clear about the actual behavior in case the user chooses to ignore the warning.

Reply:
OK - done